pkg/errors/catcher.go

```diff
@@ -303,6 +303,24 @@ func CatchServiceNameInUseError(err error, r *http.Response, serviceName string)
 	return err
 }
 
+func CatchPaymentRequiredError(err error, httpResp *http.Response) error {
+	if err == nil {
+		return nil
+	}
+
+	if httpResp == nil {
+		return err
+	}
+
+	errorPayment := CatchCCloudV2Error(err, httpResp)
+	if strings.Contains(errorPayment.Error(), "402 Payment Required") {
+		message := strings.TrimSpace(errorPayment.Error()[0 : len(errorPayment.Error())-len(": 402 Payment Required")])
```

Comment:
Try https://pkg.go.dev/strings#TrimSuffix?

Reply:
had a look at this and used it here. It was a good idea to use this! But this entire part of code will not be needed anymore i guess